hystrix/settings.go

```diff
@@ -101,3 +101,15 @@ func getSettings(name string) *settings {
 
 	return s
 }
+
+func GetCircuitSettingsMap() map[string]*Settings {
```

Comment:
the word `map` in the function name is redundant, since that is apparent from the return type.